testing/endtoend/components/validator.go

```diff
@@ -264,6 +264,9 @@ func (v *ValidatorNode) Start(ctx context.Context) error {
 			fmt.Sprintf("--%s=%d", flags.InteropStartIndex.Name, offset),
 		)
 	}
+	if v.config.UseBuilder {
+		args = append(args, fmt.Sprintf("--%s", flags.EnableBuilderFlag.Name))
```

Comment:
might be better to put this settingin func createProposerSettingsPath(pubkeys []string, nodeIdx int) (string, error) {
https://docs.prylabs.network/docs/execution-node/fee-recipient#advanced-configure-mev-builder-and-gas-limit
if the builder is supposed to work with lighthouse too , lighthouse will probably need something similar,.

Reply:
The builder has currently been tested to only work with prysm which is why its only there for now. Not sure I understand, why is that function the place to keep the flag in ?

Reply:
it wouldn't be the flag it would just be part of the settings , this function builds the propose-settings-file. I think it should be used instead of the suggested fee recipient. on our e2e runs since I check the validator settings. you can control some validator being builder settings while others are not in this case.

Reply:
not a big deal though could be a future improvement

Reply:
Yeah I think it would be better for a future improvement, adding in builder settings to the proposer config json would require a rewrite of the eval.